applications/main/infrared/scenes/infrared_scene_universal.c

```diff
@@ -21,18 +22,35 @@ void infrared_scene_universal_on_enter(void* context) {
         SubmenuIndexUniversalTV,
         infrared_scene_universal_submenu_callback,
         context);
+
     submenu_add_item(
         submenu,
         "Audio Players",
         SubmenuIndexUniversalAudio,
         infrared_scene_universal_submenu_callback,
         context);
+
+    submenu_add_item(
+        submenu,
+        "Projectors",
+        SubmenuIndexUniversalProjector,
+        infrared_scene_universal_submenu_callback,
+        context);
+
+    submenu_add_item(
```

Comment:
Fans are clearly shouldn't be part of this PR.

Reply:
Yep, missed out on it as I did this on my own copy which I had tampered on